### PR TITLE
Added additional information to Moderate Product Reviews Page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to Moderate Product Reviews Page

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/magento-product-reviews-moderate.html

### CODE REVIEW

File: src/marketing/magento-product-reviews-moderate.md [hunk 1]
```diff
@@ -17,13 +19,30 @@ For Commerce Product Reviews, a submitted product review must be approved before
    {:.bs-callout-info}
    Reviews with the `Pending` and `Not Approved` statuses are not displayed on the storefront.
 
+1. If applicable, set the **Visibility** of a product review for appearing in different store views.
```
Comment (dobooth) [inline src/marketing/magento-product-reviews-moderate.md:22]: "Visibility" is not in the screen shot. Does it need a refresh?

Comment (DmitryTsymbal) [reply src/marketing/magento-product-reviews-moderate.md:20]: @dobooth , thank you for noticing.

"Visibility" is appeared only if there are a few store views f.e. That is why I have added "If applicable"

Seems like on the provided screenshot, there is only one website/store/store view.

Of course we can edit an image, where the "Visibility" field will be present.

For example this one.

Screenshot 2022-01-14 at 16 39 15


Thanks!

**Review by dobooth — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by DmitryTsymbal** [comment]
I'm going to close this one, as I have created a new PR from re-forked repo, to avoid an errors during the import.
New PR: https://github.com/magento/merchdocs/pull/1688
Thanks!

**Pull request closed without merging**